packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -7772,6 +7772,68 @@ QUnit.module('label integration', () => {
 });
 
 QUnit.module('accessibility', () => {
+    QUnit.test('Tag aria-label should be tag text value not index', function(assert) {
+        const productsData = [
+            'HD Video Player',
+            'SuperHD Video Player',
+            'SuperPlasma 50',
+        ];
+        const productLabel = { 'aria-label': 'Product' };
+        const tagBox = $('#tagBox').dxTagBox({
+            items: productsData,
+            inputAttr: productLabel,
+            value: [...productsData],
+            maxDisplayedTags: 3,
+            showMultiTagOnly: false
+        }).dxTagBox('instance');
+        const simpleTags = tagBox.$element().find(`.${TAGBOX_TAG_CLASS}`);
+
+        assert.strictEqual($(simpleTags[0]).attr('aria-label'), 'HD Video Player', 'aria-label is tagged correctly');
+        assert.strictEqual($(simpleTags[1]).attr('aria-label'), 'SuperHD Video Player', 'aria-label is tagged correctly');
+        assert.strictEqual($(simpleTags[2]).attr('aria-label'), 'SuperPlasma 50', 'aria-label is tagged correctly');
+    });
+
+    QUnit.test('Tags aria labels should match inputAttr aria-label mappings', function(assert) {
+        const productsData = [{
+            ID: 1,
+            Name: '1',
```

Comment:
Let's use text for Name property, for example `Name: 'Tag_1'` for the first item. In this case I expect that your asserts will as following:
```
assert.strictEqual($(simpleTags[0]).attr('aria-label'), 'Tag_1', 'aria-label is tagged correctly');
```

Reply:
[07a3b33](https://github.com/DevExpress/DevExtreme/pull/27157/commits/07a3b333eec2215ee5afb0c0444a8f547a6a2f61)